coc.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Code of Conduct
+categories: nav
+showinnav: true
+---
+
+# {{ page.title }}
+
+## Austin Hackers Association (AHA!) Code of Conduct
+
+Welcome to AHA! If you’re here, you already know this isn’t your typical buttoned-up tech meetup. We swear, we heckle, and we’re all just here to have a great f\*\\*\*ing time. But even in our chaotic little corner, we have some ground rules.
+
+#### Roast the Ideas, Not the Person
+
+>> Got a hot take? Bring it. Got a dumb take? Expect to hear about it. But keep it fun. Don’t be a d\*\*k to the person; they’re here to learn and blow off steam just like you.
+
+#### No Creeps, No Bullies, No Exceptions
+
+>> We don’t f\*\* around with harassment, discrimination, or any of that other crap. If you make someone feel uncomfortable based on who they are, you’re out. Full stop. Play nice or don’t play here.
+
+#### Keep Our St Here\*\*
```

Comment:
If we're censoring for humor purposes, then be consistent

This line should be:
`#### Keep Our S\*\*t Here`